src/signalrclient/websocket_transport.h

```diff
@@ -46,6 +46,7 @@ namespace signalr
         signalr_client_config m_signalr_client_config;
 
         std::shared_ptr<cancellation_token_source> m_receive_loop_cts;
```

Comment:
We think this could be replaced with a bool and a lock since nothing registers with this. It makes it clearer this is for tracking state and not signaling anything.